src/main/groovy/org/mockito/release/gradle/BumpVersionFileTask.java

```diff
@@ -38,7 +38,10 @@ public void setVersionFile(File versionFile) {
     /**
      * See {@link BumpVersionFileTask}
      */
-    @TaskAction public void bumpVersionFile() {
+    @TaskAction public VersionInfo bumpVersionFile() {
+        if(!versionFile.exists()){
```

Comment:
>I'm a little confused because you tweaked the thrown exception's message.

Looks like I've been doing mechanical changes with only half of my brain thinking :) Yes, this can be removed because it will never happen.